src/webViewInterface.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  trackPageViewEvent,
+  trackScreenViewEvent,
+  trackSelfDescribingEvent,
+  trackStructuredEvent,
+} from './tracker';
+import type { ScreenViewProps, SelfDescribing, StructuredProps } from './types';
+
+interface FullPageViewEvent {
+  title?: string | null;
+  url?: string;
+  referrer?: string;
+}
+
+type WebViewMessage = {
```

Comment:
I'm happy to do that but just want to point out that this is an internal type, it shouldn't be exposed anywhere. I didn't add it to `types.ts` because I thought that contains the public types. But if that makes more sense, I can move it over there.

Reply:
On second thought, I moved it to `types.ts`, added comments to indicate that it is an internal type.